bee-ternary/Cargo.toml

```diff
@@ -14,6 +14,7 @@ homepage = "https://www.iota.org"
 byteorder = "1.3.4"
 lazy_static = "1.4.0"
 serde = { version = "1.0", optional = true }
+num-traits = "0.2"
```

Comment:
Do you mind ordering the deps alphabetically :) ?

Reply:
Sure, will do this